TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlay/Trajectories.kt

```diff
@@ -0,0 +1,115 @@
+package org.firstinspires.ftc.teamcode.PowerPlay
+
+import com.acmerobotics.roadrunner.geometry.Pose2d
+import com.acmerobotics.roadrunner.geometry.Vector2d
+import org.firstinspires.ftc.teamcode.trajectorysequence.TrajectorySequence
+
+data class TrajectorySet(
+    val scoringPos: Pose2d,
+    val navigateTrajectory: TrajectorySequence,
+    val parkTrajectory: TrajectorySequence,
+)
+
+data class TrajectoryParameters(
+    val side: InitialSide,
+    val foundTag: Int,
+)
+
+class Trajectories(
+    drive: SparkyMecanumDrive
+) {
+
+    private val navigationTrajectoryMap: MutableMap<InitialSide, Pair<TrajectorySequence, Pose2d>> =
+        mutableMapOf()
+    private val trajectoryMap: MutableMap<TrajectoryParameters, TrajectorySet> = mutableMapOf()
+
+    init {
+        val possibleTags = listOf(
+            PowerPlayAutoRoadRunner.TAG_1,
+            PowerPlayAutoRoadRunner.TAG_2,
+            PowerPlayAutoRoadRunner.TAG_3,
+        )
+        for (side in InitialSide.values()) {
+            for (tag in possibleTags) {
+                val set = buildTrajectories(side, drive, tag, side.startingPose())
+                trajectoryMap[TrajectoryParameters(side, tag)] = set
+                navigationTrajectoryMap[side] = Pair(set.navigateTrajectory, set.scoringPos)
+            }
+        }
+    }
+
+    fun getTrajectoryOne(side: InitialSide): Pair<TrajectorySequence, Pose2d> {
```

Comment:
Suggest giving the trajectories more descriptive names than "One" and "Two".  Like "StartToScore" and "ScoreToPark" and "StartToPark".

Reply:
You gave them names in the TrajectorySet, why not make them consistent throughout?

Reply:
Only real reason was so that there was minimal impact in the RoadRunnerAuto, but if we're down to change it then I think we should.